helm/cluster-cloud-director/values.schema.json

```
@@ -32,12 +32,12 @@
                 "ubuntu-2004-kube-v1.22.5"
             ]
         },
-        "diskSize": {
-            "type": "string",
-            "title": "Node disk size",
+        "diskSizeGB": {
+            "type": "integer",
+            "title": "Node disk size in GB",
             "description": "Change size of the nodes OS disk.",
             "examples": [
-                "30Gi"
+                "30"
```

Integer instead of string

```suggestion
                30
```